README.md

```diff
@@ -1,2 +1,36 @@
-# Musical-instrumentals-website
+## Musical-instrumentals-website
 Jump on the Beat
+
+Desktop and Mobile version of Project Musical-instrumentals-website:
```

Comment:
- [ ] Please make sure that a link to the deployed project is present in the README. You should use Github pages to deploy the project.